datasets/xtreme_s/xtreme_s.py

```diff
@@ -0,0 +1,981 @@
+# coding=utf-8
+# Copyright 2022 The Google and HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import glob
+import os
+import random
+import subprocess
+import time
+import warnings
+from ast import literal_eval
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import pandas as pd
+from tqdm import tqdm
+
+import datasets
+from datasets.tasks import AutomaticSpeechRecognition
+
+
+logger = datasets.logging.get_logger(__name__)
+
+
+""" XTREME-S Dataset"""
+
+"""TODO(PVP): Add a description here."""
+
+# TODO(PVP): BibTeX citation
+_CITATION = """\
+"""
+
+# TODO(PVP): Correct later
+_DESCRIPTION = """\
+The Cross-lingual TRansfer Evaluation of Multilingual Encoders for Speech (XTREME-S) benchmark is a benchmark designed to evaluate speech representations across languages, tasks, domains and data regimes. It covers XX typologically diverse languages eight total downstream tasks grouped in four families: speech recognition, translation, classification and retrieval.
+"""
+
+_ID_TO_LANG = {
+    "en": "english",
+    "de": "german",
+    "nl": "dutch",
+    "fr": "french",
+    "es": "spanish",
+    "it": "italian",
+    "pt": "portuguese",
+    "pl": "polish",
+}
+
+_BABEL_LANG_TO_ID = {
+    "as": "IARPA_BABEL_OP1_102_LDC2016S06",
+    "ka": "IARPA_BABEL_OP3_404_LDC2016S12",
+    "lo": "IARPA_Babel_203_LDC2017S08",
+    "sw": "IARPA_BABEL_OP2_202_LDC2017S05",
+    "tl": "IARPA_BABEL_BP_106_LDC2016S13",
+}
+
+
+_BABEL_LANG = list(_BABEL_LANG_TO_ID.keys())
+_MLS_LANG = sorted(["nl", "en", "fr", "de", "it", "pl", "pt", "es"])
+_VOXPOPULI_LANG = sorted(["en", "de", "fr", "es", "pl", "it", "ro", "hu", "cs", "nl", "fi", "hr", "sk", "sl"])
+
+# fmt: off
+_COVOST2_TO_EN_LANG = [
+    f"{source}.en"
+    for source in [
+        "fr", "de", "es", "ca", "it", "ru", "zh", "pt", "fa", "et", "mn", "nl", "tr", "ar", "sv", "lv", "sl", "ta", "ja", "id", "cy",
+    ]
+]
+_COVOST2_FROM_EN_LANG = [
+    f"en.{transcription}"
+    for transcription in [
+        "de", "ca", "zh", "fa", "et", "mn", "tr", "ar", "sw", "lv", "sl", "ta", "ja", "id", "cy",
+    ]
+]
```

Comment:
```suggestion
_COVOST2_TO_EN_LANG = [
    f"{source}.en"
    for source in [
        "fr", "de", "es", "ca", "it", "ru", "zh-CN", "pt", "fa", "et", "mn", "nl", "tr", "ar", "sv-SE", "lv", "sl", "ta", "ja", "id", "cy",
    ]
]
_COVOST2_FROM_EN_LANG = [
    f"en.{transcription}"
    for transcription in [
        "de", "ca", "zh-CN", "fa", "et", "mn", "tr", "ar", "sv-SE", "lv", "sl", "ta", "ja", "id", "cy",
    ]
]
```

Reply:
Thanks! Updating on the google/xtreme-s dataset

Reply:
done